### ISSUE TITLE
Why do we have WARNING: No version of Node.js specified in package.json, see:

### ISSUE THREAD

**Issue by peterheard01** [linemanjs/heroku-buildpack-lineman#34]
Hi Chaps,

It's deployment time, but.....

In our package.json file we have got...

  "engines": {
    "node": ">= 0.10.0"
  },

(which by the way is working successfully with another hosted lineman app for another project on heroku).

But we created a new project and are getting...

-----> Fetching custom git buildpack... done
-----> Lineman app detected
Reusing previous heroku NPM build cache
-----> Resolving engine versions

```
   WARNING: No version of Node.js specified in package.json, see:
   https://devcenter.heroku.com/articles/nodejs-support#versions
```

/tmp/node-bootstrap_node-cg3V/bin/node: error while loading shared libraries: libssl.so.0.9.8: cannot open shared object file: No such file or directory
 !     Requested engine node version  does not match available versions: 0.10.21 0.10.20 0.10.19 0.10.18 0.10.17 0.10.16 0.10.15 0.10.14 0.10.13 0.10.12 0.10.11 0.10.10 0.10.9 0.10.8 0.10.6 0.10.5 0.10.4 0.10.3 0.10.2 0.10.1 0.10.0 0.8.26 0.8.25 0.8.24 0.8.23 0.8.22 0.8.21 0.8.19 0.8.14 0.8.13 0.8.12 0.8.11 0.8.10 0.8.8 0.8.7 0.8.6 0.8.5 0.8.4 0.8.3 0.8.2 0.8.1 0.8.0 0.6.20 0.6.18 0.6.17 0.6.16 0.6.15 0.6.14 0.6.13 0.6.12 0.6.11 0.6.10 0.6.8 0.6.7 0.6.6 0.6.5 0.6.3 0.4.10 0.4.7 

I don't get it, the version in our package.json is the same as what heroku want. Or is there something in the build pack that I should be aware of?

Thanks,
Pete

**Comment by chrisdugne** [comment]
exactly the same issue for me, using
```
  "engines": {
    "node": "0.10.x"
  },
```
I tried the command line like you @peterheard01 
```
heroku create myapp --stack cedar --buildpack https://github.com/meenie/heroku-buildpack-nodejs-grunt-bower 
```
And know it's ok...

**Comment by peterheard01** [comment]
yeah that is strange, least it wasn’t just me being stupid!

**Comment by jrhorn424** [comment]
Same error. None of these workarounds worked for me.

If I use `--stack cedar`, I can get further in the build process, but then the build fails due to cedar defaulting to ASCII instead of UTF-8. If I use the default stack, `cedar-14`, I get the error reported originally in this issue.

**Comment by searls** [comment]
@jrhorn424 I don't have time to debug this currently. Maybe your repo has an encoding issue?

**Comment by peterheard01** [comment]
Getting this problem again after being kicked off Cedar-10! how annoying....

**Comment by searls** [comment]
Yep. We haven't invested time to look into this yet, and I suspect upgraing to cedar-14 is going to be a pain for all of our lineman apps.

**Comment by peterheard01** [comment]
ah ok, well if anyone else gets this problem at least they know. For me as a workaround I….

1) lineman build
2) then drop the output in a static site 
3) and put a php called index.php file in with this in it….<?php include_once("index.html"); ?>
